saltyrtc/server/server.py

```diff
@@ -45,7 +58,10 @@
 
 
 @asyncio.coroutine
-def serve(ssl_context, key, paths=None, host=None, port=8765, loop=None):
+def serve(
+        ssl_context, key, paths=None, host=None, port=8765, loop=None,
+        event_callbacks: Dict[Event, List[Coroutine]] = None
```

Comment:
They also do it like you in [PEP 484](https://www.python.org/dev/peps/pep-0484/#id48) so fair enough.